docs/middleware/BLSRegistryCoordinatorWithIndices.md

```diff
@@ -0,0 +1,80 @@
+# BLSRegistryCoordinatorWithIndices
+
+This contract is deployed for every AVS and serves as the main entrypoint for operators to register and deregister from the AVS. In addition, it is where the AVS defines its operator churn parameters.
+
+## Flows
+
+### registerOperator
+
+When registering the operator must provide 
+1. The quorums they are registering for
+2. Their BLS public key that they registered with the [BLSPubkeyCompendium](./BLSPublicKeyCompendium.md)
+3. The socket (ip:port) at which AVS offchain actors should make requests
+
+The RegistryCoordinator then
+1. Registers their BLS public key with the [BLSPubkeyRegistry](BLSPubkeyRegistry.md) and notes the hash of their public key as their operator id
+2. Registers them with the StakeRegistry 
+3. Registers them with the IndexRegistry
```

Comment:
recommend replacing "their" with "the operator's" and "them" with "the operator". Might seem repetitive, but it's probably the most precise way to write

Reply:
Fixed cf883904